.tool-versions

```diff
@@ -1,2 +1,2 @@
-java adoptopenjdk-8.0.252+9.1
+java temurin-11.0.20+101
 maven 3.8.5
```

Comment:
I didn't know about this file.  The Maven version on ci.jenkins.io has been 3.9.x for a long time.  Would you like me to submit a separate pull request to make this 3.9.5?

Reply:
Don't bother, I'll do it. I actually wondered before updating the file and looked into the documentation where > 3.8.3 was recommended without more details. I wonder how hard it would be to have the doc reading from jenkins.io jcasc config for that piece of information.